modules/indicia_setup/db/version_8_20_0/202306021401_recording_scheme_tables.sql

```diff
@@ -0,0 +1,61 @@
+-- Table: recording_scheme
+
+-- DROP TABLE recording_scheme cascade;
+
+CREATE TABLE recording_scheme
+(
+  id serial NOT NULL,
+  scheme_key varchar PRIMARY KEY, -- Unique key for a recording scheme.
```

Comment:
An alternative name for scheme_key would be external_key (which is consistent with other parts of the Indicia schema).

Reply:
Changed scheme_key to external_key.